src/components/modal/modal.js

```diff
@@ -308,13 +314,17 @@ Modal.prototype = {
     // part of the Modal Buttonset
     if (this.settings.showCloseBtn && !this.isCAP) {
       const closeBtn = $(`
-        <button type="button" class="btn-icon btn-close" title="${Locale.translate('Close')}" aria-hidden="true">
+        <button type="button" class="btn-icon btn-close" title="${Locale.translate(this.settings.closeBtnOptions.closeBtnTooltip)}" aria-hidden="true">
```

Comment:
```suggestion
        <button type="button" class="btn-icon btn-close" title="${Locale.translate(this.settings.closeBtnOptions.closeBtnTooltip),  { showAsUndefined: false, showBrackets: false }}" aria-hidden="true">
```

I think this will fix the issue with unneeded brackets 
![Screen Shot 2021-08-05 at 9 25 51 AM](https://user-images.githubusercontent.com/814283/128357540-37d1a340-4de2-4c31-aad4-70a18443d0cd.png)